plugins/filter/merge.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
Can we rename the file `merge_with_app` ?

Reply:
My plan was to add merge-related filters in this file, not only the `merge_with_app` filter.